packages/components/src/FormatFile/FormatFile.css

```diff
@@ -77,6 +77,10 @@
   box-shadow: var(--shadow-focus);
 }
 
+.deleteButton {
+  z-index: 1;
```

Comment:
lil nitpick: is there a token value we can use here instead?

Reply:
@ZakaryH that's a good nit. I can put `elevation-base` on there and send it to you for review?

Reply:
sounds good to me!